scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
The point of `src/nvim/generators/` would the that these would be invoked by `src/nvim/CMakeLists.txt`, take files in `src/nvim/` as input,  and they also could all take `{root}/src/nvim` as first argument as library path. But I'm not really sure.